.github/workflows/unit-test.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        node: [12, 14, 16]
+        node: [14, 16]
```

Comment:
.github/workflows/integration-test.yml has only 16. Are we doing integration tests only 16 and unit 14 and 16 ?